contact.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+      <meta charset="UTF-8" />
+      <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+      <link rel="stylesheet" href="/src/output.css" />
+      <title>Contact</title>
+   </head>
+   <body class="font-outfit text-webGreen">
+      <!-- Navbar -->
+      <header>
+         <nav
+            class="top-0 z-10 sticky flex flex-row justify-between items-center lg:mb-12 md:px-8 p-4 lg:p-6 xl:p-6 font-outfit"
+         >
+            <div class="flex flex-col md:text-2xl lg:text-2xl">
+               <p>DE</p>
+               <p>EN</p>
+            </div>
+            <div class="lg:absolute lg:pl-[40%]">
+               <h1 class="font-semibold text-2xl md:text-5xl">Baumeister</h1>
+            </div>
+            <div
+               class="absolute lg:flex flex-col lg:items-end space-y-10 md:space-y-24 lg:space-y-0 hidden md:ml-[20%] lg:ml-0 pt-[120%] md:pt-[70%] lg:pt-14 pl-32 lg:pl-[87%] font-bold md:font-bold lg:font-normal text-3xl md:text-5xl lg:text-2xl"
```

Comment:
Same as here, font-bold will also work on the md(medium) screens you can remove this md:font-bold